File: C#/RefactoringWorkshop/RefactoringWorkshop/Coupler/MessageChain/MetadataCodes.cs
using System.Collections.Generic;

namespace RefactoringWorkshop.Coupler.MessageChain
{
    public class MetadataCodes
    {
        private Dictionary<string, string> _companyCodes = new Dictionary<string, string>();

        public Dictionary<string, string> GetCompanyCodes() {
            return _companyCodes;
        }

        public void AddCompanyCode(string code, string company) {
            _companyCodes.Add(code, company);
        }
    }
}

File: Java/RefactoringWorkshop/src/main/java/coupler/messagechain/MetadataCodes.java
package coupler.messagechain;

import java.util.HashMap;
import java.util.Map;

public class MetadataCodes {
    private Map<String, String> companyCodes = new HashMap<>();

    public Map<String, String> getCompanyCodes() {
        return this.companyCodes;
    }

    public void addCompanyCode(String code, String company) {
        this.companyCodes.put(code, company);
    }
}
